test/test_e2e_asr_transducer.py

```diff
@@ -249,7 +249,7 @@ def test_pytorch_transducer_trainable_and_decodable(train_dic, recog_dic):
 
 
 @pytest.mark.skipif(not torch.cuda.is_available(), reason="gpu required")
-@pytest.mark.parametrize("trans_type", ["warp-transducer", "warp-rnnt"])
+@pytest.mark.parametrize("trans_type", ["warp-transducer", "warp-rnnt", "torchaudio"])
```

Comment:
Currently torchaudio throws the following error:
```
Fatal Python error: Aborted

Current thread 0x00007fd7645d6740 (most recent call first):
  File "/data/home/carolinechen/espnet/tools/venv/lib/python3.8/site-packages/torchaudio/prototype/rnnt_loss.py", line 103 in forward
  File "/data/home/carolinechen/espnet/tools/venv/lib/python3.8/site-packages/torchaudio/prototype/rnnt_loss.py", line 169 in rnnt_loss
  File "/data/home/carolinechen/espnet/tools/venv/lib/python3.8/site-packages/torchaudio/prototype/rnnt_loss.py", line 224 in forward
  File "/data/home/carolinechen/espnet/tools/venv/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1051 in _call_impl
  File "/data/home/carolinechen/espnet/espnet/nets/pytorch_backend/transducer/loss.py", line 81 in forward
  File "/data/home/carolinechen/espnet/tools/venv/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1051 in _call_impl
  File "/data/home/carolinechen/espnet/espnet/nets/pytorch_backend/e2e_asr_transducer.py", line 427 in forward
  File "/data/home/carolinechen/espnet/tools/venv/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1051 in _call_impl
...
```

Reply:
Do you know the exact line where this error got triggered?